client/src/components/TagField.js

```diff
@@ -231,6 +242,10 @@ TagField.defaultProps = {
   lazyLoad: false,
   creatable: false,
   multi: false,
+  SelectComponent: Select,
```

Comment:
Allowing passing in components to because we're no longer doing shallow() renders and the react-select components don't want to simply render, instead they throw all sorts of errors on render() including `Warning: Invalid hook call. Hooks can only be called inside of the body of a function component.`

Reply:
Please also add prop types for these.

Reply:
Updated